src/test/__tests__/components/EditControlPane.test.tsx

```diff
@@ -36,20 +35,24 @@ describe('EditControlPane', () => {
     const getMarcRecordMock = (jest.spyOn(recordsApi, 'getMarcRecord') as any).mockImplementation(() =>
       Promise.resolve(null),
     );
-    
+
     const { findByText, findByTestId } = renderWrapper();
 
-    fireEvent.click(await findByTestId('edit-control-actions-toggle'));
-    fireEvent.click(await findByText('ld.viewMarc'));
+    await act(async () => {
+      fireEvent.click(await findByTestId('edit-control-actions-toggle'));
+      fireEvent.click(await findByText('ld.viewMarc'));
+    });
 
     expect(getMarcRecordMock).toHaveBeenCalled();
   });
 
-  test('handles duplicate resource navigation', async () => {    
+  test('handles duplicate resource navigation', async () => {
     const { findByText, findByTestId } = renderWrapper();
 
-    fireEvent.click(await findByTestId('edit-control-actions-toggle'));
-    fireEvent.click(await findByText('ld.duplicate'));
+    await act(async () => {
```

Comment:
Isn’t it recommended to use `find…` queries or `waitFor` wrapper instead of `act`?